crates/red_knot_python_semantic/resources/mdtest/generics.md

```diff
@@ -16,10 +16,11 @@ class MyBox[T]:
     def __init__(self, data: T):
         self.data = data
 
-# TODO not error (should be subscriptable)
-box: MyBox[int] = MyBox(5)  # error: [non-subscriptable]
-# TODO error differently (str and int don't unify)
-wrong_innards: MyBox[int] = MyBox("five")  # error: [non-subscriptable]
+box: MyBox[int] = MyBox(5)
+
+# TODO should emit a diagnostic here (str and int don't unify)
```

Comment:
If we're editing this message anyway, let's use language from the Python typing spec instead of Hindley-Milner language
```suggestion
# TODO should emit a diagnostic here (str is not assignable to int)
```